src/app/shell/personal-cabinet/shared-cabinet/messages/chat/chat.component.ts

```diff
@@ -220,7 +220,12 @@ export class ChatComponent implements OnInit, AfterViewInit, OnDestroy {
         readDateTime: parsedMessage.ReadDateTime
       };
 
-      this.messages.push(message);
+      if (this.chatRoom.id === message.chatRoomId) {
+        this.messages.push(message);
+      } else {
+        // do nothing
```

Comment:
Should there be any logic? 
Remove the else block if not

Reply:
fixed